html/changelogs/archive/2022-03.yml

```diff
@@ -0,0 +1,26 @@
+2022-03-12:
+  Fantii644:
+  - rscadd: Tasers are now back in the game! They can be purchased in Cargo and will be found in armory
+      (three tasers will spawn there)
+  - balance: Tasers no longer hard stamcrits on one shot, gives some minor stam damage and knockdowns
```

Comment:
It used to hardstun. Stamcrit's a different thing.

Reply:
i didnt fucking write this you clown, go over to his PR and complain about it lmao